zq_test.go

```diff
@@ -1,21 +1,23 @@
-package tests
+package zq
 
 import (
 	"os"
 	"path/filepath"
+	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/brimsec/zq/ztest"
 	"github.com/stretchr/testify/require"
 )
 
-func TestZTest(t *testing.T) {
+func TestZq(t *testing.T) {
 	t.Parallel()
-	dirs := map[string]bool{}
+	dirs := map[string]struct{}{}
+	re := regexp.MustCompile(`.*ztests/.*\.yaml$`)
 	err := filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
-		if strings.HasSuffix(path, ".yaml") {
-			dirs[filepath.Dir(path)] = true
+		if !info.IsDir() && strings.HasSuffix(path, ".yaml") && re.MatchString(path) {
+			dirs[filepath.Dir(path)] = struct{}{}
```

Comment:
re will never match on Windows because the path separator there is \. You could fix a few ways, but I think the simplest is to assume interest in any directory named ztests. Then you could just do this.
```go
	 var dirs []string
	 err := filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
		if info.IsDir() && filepath.FileName(path) == "ztests" {
			dirs = append(dirs, path)
		}
		return err
	}
```

Reply:
Just pushed a fix to match both backslash or forward slash.